### PR TITLE
Don't validate ip address for mgmt interface

### PR DESCRIPTION
**Pull request by pabelanger**
##### SUMMARY
Dont' assume mgmt interface has an ip address

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
eos_facts

### CODE REVIEW

File: test/integration/targets/eos_facts/tests/cli/default_facts.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@
       # Items from those subsets are present
       - "result.ansible_facts.ansible_net_filesystems is defined" #hw
       - "result.ansible_facts.ansible_net_memtotal_mb > 10" #hw
-      - "result.ansible_facts.ansible_net_interfaces.Management1.ipv4.masklen > 1" # interfaces
+      - "result.ansible_facts.ansible_net_interfaces.Management1" # interfaces
```
Comment (Qalthos) [inline test/integration/targets/eos_facts/tests/cli/default_facts.yaml:26]: Is there anything we _can_ assert about this?

Comment (pabelanger) [reply test/integration/targets/eos_facts/tests/cli/default_facts.yaml:26]: We can, but we should update the testing here to create the ip address first.

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 84ee713e140ce6a899deb7ddcc04549da5b6184e

**Review by justjais — APPROVED** [review]

**Review by Qalthos — APPROVED** [review]

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now f0a0d2aac9e2bdcb8b91c4856d1befc2e7e47a92

**Review by Qalthos — APPROVED** [review]

**Pull request merged**